src/lib/viewers/image/ImageViewer.js

```diff
@@ -378,6 +379,28 @@ class ImageViewer extends ImageBaseViewer {
         }
     }
 
+    loadUIReact() {
+        super.loadUIReact();
+        this.renderUI();
+    }
+
+    renderUI() {
+        if (this.zoomControls) {
```

Comment:
Could this be an `else` condition attached to below's `if`?

Reply:
I'd rather keep it as simple as possible, since much of this is temporary and will be removed once we fully convert the controls.